torch/csrc/jit/tensorexpr/cuda_random.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
+namespace torch {
+namespace jit {
+namespace tensorexpr {
+
+constexpr auto philox_random_string = R"(
```

Comment:
Is this implementation identical to `rand_support_literal` from cuda/fuser/resource_strings.h?  If so can we just #include that?

Reply:
Yes. I plan to experiment with it later. But for now, we can just include it.